lib/stable.pm

```diff
@@ -0,0 +1,103 @@
+package stable;
```

Comment:
good idea but those are still experiments, so name `experimental::stable` will be imho better